gradio-blocks/README.md

```diff
@@ -0,0 +1,100 @@
+
+Gradio can be installed via pip
+
+```pip install gradio```
+
+
+**What is Blocks?**
+
+gradio.Blocks is a low-level API that allows you to have full control over the data flows and layout of your application. You can build very complex, multi-step applications using Blocks. If you have already used gradio.Interface, you know that you can easily create fully-fledged machine learning demos with just a few lines of code. The Interface API is very convenient but in some cases may not be sufficiently flexible for your needs. For example, you might want to:
+
+Group together related demos as multiple tabs in one web app
+Change the layout of your demo instead of just having all of the inputs on the left and outputs on the right
+Have multi-step interfaces, in which the output of one model becomes the input to the next model, or have more flexible data flows in general
+Change a component's properties (for example, the choices in a Dropdown) or its visibilty based on user input
+
+to learn more about Blocks see the guide https://www.gradio.app/introduction_to_blocks/
+
+
+**How Does Gradio and Hugging Face work together?**
+
+Hugging Face Spaces is a free hosting option for Gradio demos. Spaces comes with 3 SDK options: Gradio, Streamlit and Static HTML demos. Spaces can be public or private and the workflow is similar to github repos. There are over 2000+ Gradio spaces currently on Hugging Face. Learn more about spaces [here](https://huggingface.co/docs/hub/spaces).
+
+**Event Plan**
+
+main components of the event consist of:
+
+- Learning about Gradio and the new Blocks Feature
+- How Gradio and Hugging Face Spaces work together
+- Understanding the Hugging Face ecosystem
+- Share Demos in a Hugging Face organization for the Blocks event
+
+
+**Example spaces using Blocks**
+- [https://huggingface.co/spaces/akhaliq/ArcaneGAN-blocks](https://huggingface.co/spaces/akhaliq/ArcaneGAN-blocks)
+- [https://huggingface.co/spaces/merve/gr-blocks](https://huggingface.co/spaces/merve/gr-blocks)
+- [https://huggingface.co/spaces/osanseviero/tortoisse-tts](https://huggingface.co/spaces/osanseviero/tortoisse-tts)
+- [https://huggingface.co/spaces/akhaliq/CaptchaCracker](https://huggingface.co/spaces/akhaliq/CaptchaCracker)
+
+
+To participate in the event
+
+- Join the organization for Blocks event
+    - [https://huggingface.co/Gradio-Blocks](https://huggingface.co/Gradio-Blocks)
+- Join the discord
+    - [discord](https://discord.com/invite/feTf9x3ZSB)
+
+
+Participants will have a month to build and share Gradio demos using the Blocks feature. We will share a list of ideas of spaces that can be created using blocks or participants are free to try out their own ideas. At the end of the event, spaces will be evaluated for creativity and prizes will be given 
+
+
+potential ideas for creating spaces:
```

Comment:
We should make it clear that they don't have to restrict themselves to these

Reply:
sounds good, ill update to add that

Reply:
updated